docs/integrations/emr/emr_configuration.md

```diff
@@ -0,0 +1,215 @@
+
+!!! info
+    Ensure [Networking](networking.md) was set up correctly before proceeding with this guide.
+
+## Step 1: Setting up a Hopsworks API Key
+In order for EMR clusters to be able to communicate with the Hopsworks Feature Store, the clients running on EMR need to be able to access a Hopsworks API Key.
+
+### Generating an API Key
+
+In Hopsworks, click on your *username* in the top-right corner and select *Settings* to open the user settings. Select *Api keys*. Give the key a name and select the project scope before creating the key. Copy the key into your clipboard for the next step.
+
+!!! success "Scopes"
+    The created API-Key should at least have the following scopes:
+
+    1. project
+
+<p align="center">
+  <figure>
+    <img src="../../../assets/images/emr/api_key.png" alt="Generating an API Key on Hopsworks">
+    <figcaption>API-Keys can be generated in the User Settings on Hopsworks</figcaption>
+  </figure>
+</p>
+
+!!! info
+    You are only able to retrieve the API Key once. If you miss to copy it to your clipboard, delete it again and create a new one.
+
+### Storing the API Key in the AWS Secrets Manager
+
+In the AWS management console ensure that your active region is the region you use for EMR.
+Go to the *AWS Secrets Manager* and select *Store new secret*. Select *Other type of secrets* and add *api-key*
+as the key and paste the API key created in the previous step as the value. Click next.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/databricks/aws/databricks_secrets_manager_step_1.png">
+      <img src="../../../assets/images/databricks/aws/databricks_secrets_manager_step_1.png" alt="Storing a Feature Store API Key in the Secrets Manager Step 1">
+    </a>
+    <figcaption>Storing a Feature Store API Key in the Secrets Manager Step 1</figcaption>
+  </figure>
+</p>
+
+As secret name, enter *hopsworks/featurestore*. Select next twice and finally store the secret.
+Then click on the secret in the secrets list and take note of the *Secret ARN*.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/emr/secrets_manager.png">
+      <img src="../../../assets/images/emr/secrets_manager.png" alt="Storing a Feature Store API Key in the Secrets Manager Step 2">
+    </a>
+    <figcaption>Storing a Feature Store API Key in the Secrets Manager Step 2</figcaption>
+  </figure>
+</p>
+
+### Granting access to the secret to the EMR EC2 instance profile
+
+Identify your EMR EC2 instance profile in the EMR cluster summary:
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/emr/emr_instance_profile.png">
+      <img src="../../../assets/images/emr/emr_instance_profile.png" alt="Identify your EMR EC2 instance profile">
+    </a>
+    <figcaption>Identify your EMR EC2 instance profile</figcaption>
+  </figure>
+</p>
+
+
+In the AWS Management Console, go to *IAM*, select *Roles* and then the EC2 instance profile used by your EMR clusters.
```

Comment:
cluster (not clusters) ?

Reply:
It doesn't really matter but I can say cluster for consistency. EMR clusters are meant to be ephemeral so in reality, it's clusters.